table/table.go

```diff
@@ -26,6 +26,10 @@ func (r Row) findColumnNumber(colName string) int {
 // text.Colors{} to use on the entire row
 type RowPainter func(row Row) text.Colors
 
+// IndexedRowPainter is a custom function that takes a Row index as input and returns the
+// text.Colors{} to use on the entire row
+type IndexedRowPainter func(idx int) text.Colors
+
```

Comment:
I want to work on a generic RowPainter which does both this and more instead of introducing multiple variants like this. Give me a day please.

Reply:
Sure. I implemented as a separate variant so as to not introduce a breaking change. If you don't have time, I can implement a generic version.